sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/b2c/AADB2CProperties.java

```diff
@@ -40,12 +41,6 @@
 
     public static final String PREFIX = "azure.activedirectory.b2c";
 
-    /**
-     * The name of the b2c tenant.
-     */
-    @NotBlank(message = "tenant name should not be blank")
-    private String tenant;
```

Comment:
It's a breaking change?

Reply:
yes, do you have other ways to do that?

Reply:
Next release version is minor version upgrade. So we should keep backward compatibility. How about using `@Deprecated` annotation?

Reply:
@saragluna , how about Rujun's comment? there's the same concern from xiaobing's PR